modules/nix-bitcoin.nix

```diff
@@ -107,7 +121,7 @@ in {
     };
     # Give operator access to onion hostnames
     services.onion-chef.enable = true;
-    services.onion-chef.access.operator = [ "bitcoind" "clightning" "nginx" "liquidd" "spark-wallet" "electrs" "sshd" ];
+    services.onion-chef.access.operator = [ "bitcoind" "clightning" "lnd" "nginx" "liquidd" "spark-wallet" "electrs" "sshd" ];
```

Comment:
operator still doesn't seem to have access to lnd. For that operator would need to be in the same group and have an alias (see below) to use the correct data directory.